packages/mgt-chat/src/statefulClient/useBotInfo.ts

```diff
@@ -0,0 +1,14 @@
+import { useContext, useEffect, useState } from 'react';
+import { BotInfoContext } from '../components/Context/BotInfoContext';
+
+export const useBotInfo = () => {
```

Comment:
Are we suppoed to be using this hook in this PR?

Reply:
It is used...

<img width="399" alt="image" src="https://github.com/plasne/microsoft-graph-toolkit/assets/15094473/b8d30773-e6ba-49e5-8899-481f5068c133">


If you are asking are we updating our ChatList to use these features in this PR, no. This PR is just about merging from upstream.